lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -70,13 +57,35 @@
 
   .quick-nav {
     background-color: theme-color('light');
-    margin-bottom: $baseline/2;
 
     .nav-item {
       font-size: 18px;
       color: $white;
     }
   }
+}
+#program-details-page {
```

Comment:
Nit: can you run stylelint against these changes, because it should expect a newline here.